logic/jwts.go

```diff
@@ -101,6 +102,18 @@ func CreateUserJWT(username string, networks []string, isadmin bool) (response s
 	return "", err
 }
 
+// VerifyJWT verifies Auth Header
```

Comment:
either comment or func name is incorrect

Reply:
updated func name